client/app/app.js

```diff
@@ -88,4 +88,9 @@ angular.module('app', [
       }
 
     });
+
+    $rootScope.$on('$stateChangeStart', function(event, currentRoute, previousRoute) {
+      window.scrollTo(0, 0);
+    });
+
```

Comment:
I'm a little hesitant on this guy happening at every page load. the inbox has a sticky scroll bar and I'm not sure of the implications of this. what does this do exactly?